libs/features/game-tele/src/edit-game-tele/game-tele.integration.spec.ts

```diff
@@ -0,0 +1,156 @@
+    'DELETE FROM `game_tele` WHERE (`id` = ' +
+    id +
+    ');\n' +
+    'INSERT INTO `game_tele` (`id`, `position_x`, `position_y`, `position_z`, `orientation`, `map`, `name`) VALUES\n' +
+    '(' +
+    id +
+    ", 0, 0, 0, 0, 0, '');\n";
+
+  const originalEntity = new GameTele();
+  originalEntity.id = 1;
+  originalEntity.name = 'ABC';
+
+  beforeEach(waitForAsync(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        BrowserAnimationsModule,
+        ToastrModule.forRoot(),
+        ModalModule.forRoot(),
+        GameTeleComponent, // This should typically be in declarations, but as per your instruction, it's left unchanged
+        RouterTestingModule,
+        TranslateTestingModule,
+      ],
+      providers: [
+        GameTeleHandlerService,
+        {
+          provide: SqliteService,
+          useValue: instance(mock(SqliteService)),
+        },
+      ],
+    }).compileComponents();
+  }));
+
+  function setup(creatingNew: boolean) {
+    handlerService = TestBed.inject(GameTeleHandlerService);
+    handlerService['_selected'] = `${id}`;
+    handlerService.isNew = creatingNew;
+
+    queryService = TestBed.inject(MysqlQueryService);
+    querySpy = spyOn(queryService, 'query').and.returnValue(of([]));
+
+    spyOn(queryService, 'selectAll').and.returnValue(of(creatingNew ? [] : [originalEntity]));
+
+    fixture = TestBed.createComponent(GameTeleComponent);
+    page = new GameTelePage(fixture);
+    fixture.autoDetectChanges(true);
+    fixture.detectChanges();
+  }
+
+  describe('Creating new', () => {
+    beforeEach(() => setup(true));
+
+    it('should correctly initialise', () => {
+      page.expectQuerySwitchToBeHidden();
+      page.expectFullQueryToBeShown();
+      page.expectFullQueryToContain(expectedFullCreateQuery);
+      querySpy.calls.reset();
+    });
+
+    it('should correctly update the unsaved status', () => {
+      const field = 'name';
+      expect(handlerService.isGameTeleUnsaved).toBe(false);
+      page.setInputValueById(field, 'ABC');
+      expect(handlerService.isGameTeleUnsaved).toBe(true);
+      page.setInputValueById(field, '');
+      expect(handlerService.isGameTeleUnsaved).toBe(false);
+    });
+
+    it('changing a property and executing the query should correctly work', () => {
+      const expectedQuery =
+        'DELETE FROM `game_tele` WHERE (`id` = 1);\n' +
+        'INSERT INTO `game_tele` (`id`, `position_x`, `position_y`, `position_z`, `orientation`, `map`, `name`) VALUES\n' +
+        "(1, 0, 0, 0, 0, 0, 'ABC');";
+
+      querySpy.calls.reset();
+
+      page.setInputValueById('name', 'ABC');
+      page.expectFullQueryToContain(expectedQuery);
+
+      page.clickExecuteQuery();
+
+      expect(querySpy).toHaveBeenCalledTimes(1);
+      expect(querySpy.calls.mostRecent().args[0]).toContain(expectedQuery);
+    });
+
+    describe('Editing existing', () => {
+      beforeEach(() => setup(false));
+
+      it('should correctly initialise', () => {
```

Comment:
same as above, but this time with `false`:

```suggestion
      it('should correctly initialise', () => {
      const { page } = setup(false);
```

and in the other `it` blocks as well